package.json

```diff
@@ -99,12 +99,13 @@
     "@github/relative-time-element": "^4.1.2",
     "@lit-labs/react": "1.1.1",
     "@primer/behaviors": "1.3.4",
-    "@primer/octicons-react": "18.3.0",
+    "@primer/octicons-react": "^19.3.0",
```

Comment:
I moved from an exact dep, to avoid version conflicts in dotcom, which this would cause eventually otherwise (if dotcom installed, for instance, 19.4.0).

Should octicons-react just be a peer dependency and not a direct dep?